app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RegistrationsController < Devise::RegistrationsController
+  def destroy
```

Comment:
This PR is only for listing keyword, not sure why we need it 🤔

Reply:
Adding the relation `has_many` brought the lights on "how do we delete keywords". The solution making more sense according to me was not to delete any keywords, but to delete users if needed. It was also the occasion to practice with soft-delete (without implementing a non-asked feature to delete keywords) :) 
But yeah, it's arguable that it should have been done earlier in the authentication PRs.

Reply:
let me know if you think I should extract it into a chore branch~ @junan

Reply:
Ok, its fine to me as it is related to current soft delete changes and changes is not big